system/HTTP/Response.php

```diff
@@ -573,6 +573,7 @@ public function redirect(string $uri, string $method = 'auto', int $code = null)
 		{
 			if (isset($_SERVER['SERVER_PROTOCOL'], $_SERVER['REQUEST_METHOD']) && $_SERVER['SERVER_PROTOCOL'] === 'HTTP/1.1')
```

Comment:
The version 1.1 is checked here. Maybe remove the `HTTP/` and get the numbers to set below.

Reply:
But the conditional forces to be 1.1...